### ISSUE TITLE
Transmitter is shown as paired but there no info about the device

### ISSUE THREAD

**Issue by qwqatest** [Faifly/xDrip#206]
Transmitter was paired with iPhone #1.  
I turned off Bluetooth on iPhone #1
Installed app on iPhone #2 and started scan (this transmitter was already paired in the phone settings before)

Transmitter is shown as paired but without any info and SN and Device Name is different
![Screenshot at Oct 16 19-31-54](https://user-images.githubusercontent.com/47991168/96287570-d4176900-0fea-11eb-82f4-d942de7022d0.png)

Log
![Screenshot at Oct 16 19-32-15](https://user-images.githubusercontent.com/47991168/96287602-df6a9480-0fea-11eb-919e-0fe4075ae136.png)

**Comment by qwqatest** [comment]
Another Log example on iPhone #2 when trying to pair without unpair from iPhone #1

![Screenshot at Nov 17 11-46-26](https://user-images.githubusercontent.com/47991168/99374128-ef67e380-28ca-11eb-8b3d-947e38604aaf.png)

**Comment by qwqatest** [comment]
@NSFriend Reproducible in another way: 
![Screenshot at Dec 01 18-18-56](https://user-images.githubusercontent.com/47991168/100770133-9c7f4780-3405-11eb-8421-fadb100ba8ba.png)

1. Try to pair with a transmitter
2. When you get pairing notification wait for 4 minutes and open it 
3. Go to the app Settings -> Transmitter and check the screen

I've tried two times and in both cases it was reproducible. But on the 5S I didn't get pairing request anymore (waited for 17 minutes, ticket https://github.com/Faifly/xDrip/issues/149) and on 6S I got the request after 5 minutes again (and worked fine after this).
